packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -829,4 +846,74 @@ describe('Transfer tests', function(this: any) {
       })
     }
   })
+
+  describe('Transfers Frozen >', () => {
+    before(restartWithCleanNodes)
+
+    for (const syncMode of syncModes) {
+      describe(`${syncMode} Node >`, () => {
+        before(`start geth on sync: ${syncMode}`, async () => {
+          try {
+            await startSyncNode(syncMode)
+          } catch (err) {
+            console.debug('some error', err)
+          }
+        })
+
+        describe('when CeloGold is frozen', () => {
+          before('ensure gold transfers are frozen', async () => {
+            await freeze('http://localhost:8545', validatorAddress, CeloContract.GoldToken)
+          })
+
+          describe('check if frozen', () => {
+            it('should be frozen', async () => {
+              const goldTokenAddress = await kit.registry.addressFor(CeloContract.GoldToken)
+              const freezer = await kit._web3Contracts.getFreezer()
+              const isFrozen = await freezer.methods.isFrozen(goldTokenAddress).call()
+              assert(isFrozen)
+            })
+          })
+          describe('when neither sender nor receiver is whitelisted', () => {
+            it('should not add the transaction to the pool', async () => {
+              const gas = INTRINSIC_TX_GAS_COST + ADDITIONAL_INTRINSIC_TX_GAS_COST
+              const feeCurrency = await kit.registry.addressFor(CeloContract.StableToken)
+              try {
+                const res = await transferCeloGold(FromAddress, ToAddress, TransferAmount, {
+                  gas,
+                  feeCurrency,
+                })
+                await res.getHash()
+                assert.fail('no error was thrown')
+              } catch (error) {
+                assert.include(error.toString(), 'Returned error: transfers are currently frozen')
```

Comment:
I can't find this string in our repos "transfers are currently frozen". Can you point me to where it's generated?

Reply:
Yea that's added in my other PR in celo-blockchain https://github.com/celo-org/celo-blockchain/pull/890